### DISCUSSION TITLE
https://maxkim-j.github.io/posts/function-props-vuejs

### DISCUSSION THREAD

**Discussion by MaxKim-J** [MaxKim-J/maxkim-j.github.io#89] [Announcements]
#          📩 Vue.js에서 함수를 props로 넘길 수 있을까?(event emit vs function props)      

https://maxkim-j.github.io/posts/function-props-vuejs

**Comment by MaxKim-J** [comment]
<p>좋은 글 감사합니다</p>

>_This comment was made by Disqus. 2021-03-24 07:40:21 김현태(disqus_UuGoKZA8UD)_

**Comment by MaxKim-J** [comment]
<p>안티패턴이라고들 하더라구요.</p><p>굳이 안티패턴이라고 할거까지는 없다는게..<br>function 넘기는거랑 object 넘기는거랑 뭐가 다르지?</p><p>$emit 으로 처리했을시, 비동기 처리 완료가 된 후 반영은??</p>

>_This comment was made by Disqus. 2021-08-31 04:17:08 라이언(disqus_Bam85l2bkM)_

**Comment by ktseo41** [comment]
좋은 글 잘봤습니다.! 감사합니다.
